onset_maproom/config-nma-ethiopia.yaml

```diff
@@ -1,13 +1,31 @@
+# App
 mode: debug  # debug, devel or prod
 prefix: https://iridl.ldeo.columbia.edu
 core_path: /onset-maproom
 admin_path: /onset-maproom-admin
 tile_path: /onset-maproom-tiles
-daily_rainfall_path: /data/aaron/ethiopia-rain-rechunked
-rr_mrg_nc_path: /Data/data23/NMA_Ethiopia_v7/ALL_NEW/Rainfall/daily/
-rr_mrg_zarr_path: /data/remic/mydatafiles/Ethiopia/NMA_Ethiopia_v7/ALL_NEW/Rainfall/daily/rr_mrg_ALL/
 support_email: help@iri.columbia.edu
 loglevel: info
 listen_address: 127.0.0.1
 listen_port: 8050
 #Jeff set up host 0.0.0.0 port 8000 to run app on  http://shortfin01.iri.columbia.edu:8000/onset-maproom/ for demos
+
+# ENACTSvZARR
+daily_rainfall_path: /data/aaron/ethiopia-rain-rechunked
+
+# Zarr conversion
+rr_mrg_nc_path: /Data/data23/NMA_Ethiopia_v7/ALL_NEW/Rainfall/daily/
+rr_mrg_zarr_path: /data/remic/mydatafiles/Ethiopia/NMA_Ethiopia_v7/ALL_NEW/Rainfall/daily/rr_mrg_ALL/
+
+# Ethiopia
+logo: Ethiopia_IRI_98x48.png
+institution: NMA
+
+# Climate and Ag
+
+# Onset_and_Cessation
+onset_and_cessation_title: Historical Onset and Cessation Date
+default_search_month: Jun
+default_running_days: 5
+default_min_rainy_days: 3
+isonoff_cess_date_hist: inline-block
```

Comment:
This variable's name is misleading. From the name, I expected it to take a Boolean value, but it seems it's actually a string to be used as the value of an HTML `display` attribute. This could be fixed in either direction, but I recommend making it a Boolean, and letting the code fill in either `none` or `inline-block` depending on that Boolean value. That gives us the flexibility to change the mechanism by which components get shown/hidde, without having to change all the countries' config files.